onadata/apps/logger/models/instance.py

```diff
@@ -79,6 +80,9 @@ def _get_attachments_from_instance(instance):
         attachment['id'] = a.id
         attachments.append(attachment)
```

Comment:
Does the `attachment` value within the `attachments` list update if we set `attachment['deleted_at']` below ?

Reply:
Sure does!
I mean, the submission does still have all the attachments it received, but at this point, only surfaces those which havent been 'deleted'.

Reply:
Cool 👍